src/sentry/static/sentry/app/views/settings/account/apiTokens.jsx

```diff
@@ -76,6 +79,11 @@ class ApiTokens extends AsyncView {
     return (
       <div>
         <SettingsPageHeader title="Auth Tokens" action={action} />
+        <AlertLink to={`/settings/${organization.slug}/developer-settings/new-internal`}>
```

Comment:
This line seems to be breaking things in TravisCI @scefali, I could patch it with `${(organization && organization.slug) || ''` but is there something else I'm missing that would cause it to break?

Reply:
Addressed above :)